test/unit/services/instanceAdapter_spec.js

```diff
@@ -22,11 +22,29 @@ describe("The InstanceAdapter class ", function () {
 
 			instances.createInstance()
 			.then(function (result) {
+				var failedInstance = new Instance({
```

Comment:
This was used to check whether `updateInstance()` is called to change the `state` to failed in case the AWS SDK fails.